src/PostgREST/Config.hs

```diff
@@ -168,6 +169,9 @@ toText conf =
         secret = fromMaybe mempty $ configJwtSecret c
     showSocketMode c = showOct (configServerUnixSocketMode c) mempty
 
+    innerJoin JTInner = "inner"
+    innerJoin JTLeft  = "left"
```

Comment:
This should be `dumpJoin` or so, looking at the other values. But Steve is going to remove it soon anyway, so maybe not that important.

Reply:
True, fixed!